.trunk/trunk.yaml

```diff
@@ -41,8 +41,8 @@ lint:
       commands:
         - name: lint
           run:
-            cargo clippy --message-format json --locked --all-targets
-            --all-features -- --cap-lints=warn --no-deps
+            cargo clippy --workspace --exclude keth-run --message-format json
```

Comment:
keth-run is skipped from clippy because it currently panic (due to its build script looking for an env var).
Here's the [tracking issue ](https://github.com/kkrt-labs/keth/issues/1285) with a suggestion to make it work

Reply:
have you tried just to do what you suggest in the created issue? ie adding 

```
    environment:
      - name: VIRTUAL_ENV
         value: ""
```

instead of skipping?

Reply:
It breaks the path, cargo is no more found

```
~/dev/keth feat/isolate-run-proof-mode* 8s keth ❯ head .trunk/out/Cpppi.yaml    22:39:09
trunk_cli_version: 1.22.10
title: Binary not found. Next check run may repair installation
report:
  - message: Unable to find binary in PATH
    binary: cargo
    PATH:
      []
  - linter:
      command: |
```